tests/FunctionalTestCase.php

```diff
@@ -78,6 +78,9 @@ protected function assertSameDocuments(array $expectedDocuments, $actualDocument
 
     protected function getFeatureCompatibilityVersion()
     {
+        if (version_compare($this->getServerVersion(), '3.2.0', '<=')) {
```

Comment:
This won't catch any 3.2.x versions beyond the initial 3.2.0 release. Using `3.4.0` and `<` should work.